modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/dto/v1_0/converter/ProductConfigurationListDTOConverter.java

```diff
@@ -42,6 +44,13 @@ public ProductConfigurationList toDTO(
 		return new ProductConfigurationList() {
 			{
 				setActions(dtoConverterContext::getActions);
+				setCustomFields(
+					() -> CustomFieldsUtil.toCustomFields(
+						dtoConverterContext.isAcceptAllLanguages(),
+						CPDefinition.class.getName(),
```

Comment:
Andrea everything looks good to me except this needs to be fixed and should answer your question on why the customFields are not returned.

```
						CPConfigurationList.class.getName()
```

Reply:
it worked, thank you!